pkg/teldrvr/newrelicAPM.go

```diff
@@ -193,23 +192,23 @@ func (t *APMTransaction) Error(_ string, readCloser io.ReadCloser) error {
 // Info [NOT IMPLEMENTED]
 func (t *APMTransaction) Info(_ string, readCloser io.ReadCloser) error {
 	// max bytes available for the error message
-	InfoMsg := make([]byte, telemetry.ErrorBytesSize)
+	infoMsg := make([]byte, telemetry.DebugByteSize)
 	defer func() {
 		closeErr := readCloser.Close()
 		if closeErr != nil {
 			log.Printf("Telemetry driver newRelicAPM could not close reader while logging Info. Potential resource leak!")
 		}
 	}()
 
-	bytesRead, err := readCloser.Read(InfoMsg)
+	bytesRead, err := readCloser.Read(infoMsg)
 	if err != nil {
 		return errors.New("error while reading Debug message")
 	}
 
-	recordLog := newrelic.LogData{}
-	recordLog.Severity = "Debug"
-	recordLog.Message = string(InfoMsg[:bytesRead])
-	recordLog.Timestamp = time.Now().UnixMilli()
+	recordLog := newrelic.LogData{
+		Severity: "Info",
+		Message:  string(infoMsg[:bytesRead]),
```

Comment:
The timestamp is optional and if is not set it will be set to `int64(timeToUnixMilliseconds(time.Now()))`

`imestamp int64  // Optional: Unix Millisecond Timestamp; A timestamp will be generated if unset`